src/services/item/test/index.test.ts

```diff
@@ -1025,6 +1025,55 @@ describe('Item routes tests', () => {
         });
       });
 
+      it('Returns successfully folder items by write and admin', async () => {
+        const bob = await MEMBERS_FIXTURES.saveMember(MEMBERS_FIXTURES.BOB);
+        await saveItemAndMembership({
+          member: actor,
+          creator: bob,
+          permission: PermissionLevel.Read,
+          item: { type: ItemType.DOCUMENT },
+        });
+        const { item: itemFolder1 } = await saveItemAndMembership({
+          member: actor,
+          permission: PermissionLevel.Admin,
+          item: { type: ItemType.FOLDER },
+        });
+        const { item: itemFolder2 } = await saveItemAndMembership({
+          member: actor,
+          permission: PermissionLevel.Write,
+          item: { type: ItemType.FOLDER },
+        });
+        const { item: notAFolder } = await saveItemAndMembership({
+          member: actor,
+          creator: bob,
+          permission: PermissionLevel.Write,
+          item: { type: ItemType.APP },
+        });
+
+        const folders = [itemFolder1, itemFolder2];
+
+        const response = await app.inject({
+          method: HttpMethod.GET,
+          url: `/items/accessible`,
+          query: {
+            sorty: SortBy.ItemCreatorName,
+            ordering: 'asc',
+            permissions: [PermissionLevel.Write, PermissionLevel.Admin],
+            types: [ItemType.FOLDER],
+          },
+        });
+
+        expect(response.statusCode).toBe(StatusCodes.OK);
+
+        const { data, totalCount } = response.json();
+        expect(totalCount).toEqual(folders.length);
+        expect(data).toHaveLength(folders.length);
+        folders.forEach((folder, idx) => {
+          expectItem(data[idx], folder);
```

Comment:
Does this rely on the creation order ? So it will break if we somehow change how we order elements ?

Reply:
Yes it fails if the order is not respected... Maybe you're right, I will remove this control in this test case because it is not what it should test...